Solutions/Windows Security Events/Hunting Queries/GroupAddedToPrivlegeGroup.yaml

```diff
@@ -54,4 +53,15 @@ query: |
   GroupAddition
   ) on GroupSid
   | extend timestamp = GroupCreateTime, AccountCustomEntity = Account, HostCustomEntity = Computer
+  | extend NTDomain = tostring(split(AccountCustomEntity,'\\',0)[0]), Name = tostring(split(AccountCustomEntity,'\\',1)[0])
```

Comment:
Modify this line to use the original field name which is Account.
Additionally, change from using FullName for Host entityType.  You will want Name and DnsDomain or Name and NTDomain as documented here - https://learn.microsoft.com/en-us/azure/sentinel/entities-reference#host